changelog.d/5-internal/reuse-http2

```diff
@@ -1 +1 @@
-Reuse HTTP2 connections from brig, galley, cargohold and federator
\ No newline at end of file
+Reuse HTTP2 connections from brig, galley, cargohold and federator (#3120, #3233)
```

Comment:
Note to reviewer, in the previous PR I made a grammatical mistake by saying "from ... and", I meant to say "
from ... to", but with this PR the mistake is useful.